docs/build-from-scratch.md

```diff
@@ -1,67 +1,68 @@
-# Build From Scratch
+# Build from source
 
-While the [README][0] pulls from the Heptio image registry, you can also build your own Heptio Ark container with the following steps:
+* [Prerequisites][1]
+* [Download][2]
+* [Build][3]
+* [Test][12]
+* [Run][7]
+* [Vendoring dependencies][10]
 
-* [0. Prerequisites][1]
-* [1. Download][2]
-* [2. Build][3]
-* [3. Test][12]
-* [4. Run][7]
-* [5. Vendoring dependencies][10]
+## Prerequisites
 
-## 0. Prerequisites
+* Access to a Kubernetes cluster, version 1.7 or later. Version 1.7.5 is required to run `ark backup delete`.
```

Comment:
Same comment from elsewhere - "1.7.5 or higher", in case someone jumps straight to this doc.